### DISCUSSION TITLE
libmesh update (miniconda) and error in RelationshipManager.h

### DISCUSSION THREAD

**Discussion by mcacace** [idaholab/moose#16220] [Q&A Getting Started] [answered]
Dear all,
just update my conda environment, and the framework. However, on compiling I get an error as attached. 
Could anyone help? 
Thanks,
mauro
![error_libmesh](https://user-images.githubusercontent.com/13695024/99372668-ccd0cd00-28c0-11eb-8eae-833def156782.png)

**Comment by mcacace** [comment]
As an update, I tried to deactivate and (re)activate the moose env. Now things are getting worse. While trying to compile moose, I keep receiving the following error from libmesh (path to contrib/lib/libtool): 
usr/bin/sed: not found
:   error: Failed to create '/.libs'

I am kind of lost now, and any suggestion is welcome.
Thanks in advance for any help,
mauro

**Comment by milljm** [comment, marked as answer]
This is being discussed here: #16123

(the sed part)

**Reply by mcacace** [reply]
thanks

**Reply by milljm** [reply]
Not sure if I should be given credit... I do not believe your issue is related to the `sed` issue.  I read 'libmesh update (miniconda)' and assumed the sed issue. It's possible your getting further than others through the use of ccache. _maybe_.
